src/server/components/dashboard/listsItems/getViewModel.ts

```diff
@@ -105,27 +101,35 @@ function parseValue<T extends KeyOfJsonData>(
       jsonData["address.secondLine"]?.trim() ?? "",
```

Comment:
A issue has been observed here during testing.  Stacktrace received when the address.secondLine has been updated:

`(node:60844) UnhandledPromiseRejectionWarning: TypeError: _jsonData$addressSec2.trim is not a function
    at parseValue (/Users/alisalaman/dev/lists-3/dist/server.js:3235:411)
    at rowFromField (/Users/alisalaman/dev/lists-3/dist/server.js:3243:15)
    at /Users/alisalaman/dev/lists-3/dist/server.js:3277:12
    at Array.map (<anonymous>)
    at jsonDataAsRows (/Users/alisalaman/dev/lists-3/dist/server.js:3276:17)
    at getContactRows (/Users/alisalaman/dev/lists-3/dist/server.js:3290:10)
    at getDetailsViewModel (/Users/alisalaman/dev/lists-3/dist/server.js:3323:13)
    at _callee$ (/Users/alisalaman/dev/lists-3/dist/server.js:2541:91)
    at tryCatch (/Users/alisalaman/dev/lists-3/node_modules/regenerator-runtime/runtime.js:63:40)
    at Generator.invoke [as _invoke] (/Users/alisalaman/dev/lists-3/node_modules/regenerator-runtime/runtime.js:294:22)
(node:60844) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 6)
`

For some reason the address.secondLine is received as an array of values.  It looks like it maybe splitting the characters into an array somewhere in the ingest controller perhaps.  You can see what's captured from my debugger output below but it seems this line is what's causing the error.  

![Screenshot 2023-01-13 at 07 12 34](https://user-images.githubusercontent.com/79845659/212265245-839ba8c1-30a8-4a6b-8bd8-0fdc874bfa11.png)

You can (kind of) see the data that was submitted from the form runner below. The key bit which looks fine:

`{\"key\":\"address.secondLine\",\"title\":\"Address line 2\",\"type\":\"text\",\"answer\":\"XXX\"},`

Full details:

`apply_1     | {"level":"info","time":1673593627145,"pid":73,"hostname":"b74ee8f50da8","0":"WebhookService","1":"postRequest body","msg":"{\"name\":\"Digital Form Builder - Runner translators-interpreters\",\"questions\":[{\"question\":\"Which country list do you want to be added to?\",\"fields\":[{\"key\":\"country\",\"title\":\"Country list\",\"type\":\"list\",\"answer\":\"Poland\"}]},{\"question\":\"Can you provide translation or interpretation services to customers in English?\",\"fields\":[{\"key\":\"speakEnglish\",\"title\":\"English language service\",\"type\":\"list\",\"answer\":true}]},{\"question\":\"Are you a member of any professional associations?\",\"fields\":[{\"key\":\"memberOfProfessionalAssociations\",\"title\":\"Member of any associations?\",\"type\":\"list\",\"answer\":true}]},{\"question\":\"Which associations are you a part of? \",\"fields\":[{\"key\":\"regulators\",\"title\":\"Associations, unions or regulatory bodies\",\"type\":\"text\",\"answer\":\"BLAH\"}]},{\"question\":\"What size is your company or firm?\",\"fields\":[{\"key\":\"size\",\"title\":\"Size of company or firm\",\"type\":\"list\",\"answer\":\"Small firm\"}]},{\"question\":\"Contact name \",\"fields\":[{\"key\":\"contactName\",\"title\":\"Contact's name\",\"type\":\"text\",\"answer\":\"Katarzyna PawlakXXX\"}]},{\"question\":\"Company name and address\",\"fields\":[{\"key\":\"organisationName\",\"title\":\"Company name\",\"type\":\"text\",\"answer\":\"Katarzyna Pawlak TłumaczeniaXXX\"},{\"key\":\"address.firstLine\",\"title\":\"Address line 1\",\"type\":\"text\",\"answer\":\"Jana III Sobieskiego 22XXX\"},{\"key\":\"address.secondLine\",\"title\":\"Address line 2\",\"type\":\"text\",\"answer\":\"XXX\"},{\"key\":\"city\",\"title\":\"Town or city \",\"type\":\"text\",\"answer\":\"WarszawaXXX\"},{\"key\":\"postCode\",\"title\":\"Post code / area code\",\"type\":\"text\",\"answer\":\"02-930XXX\"},{\"key\":\"addressDisplay\",\"title\":\"How detailed do you want your address to appear on GOV.UK? \",\"type\":\"list\",\"answer\":\"full\"}]},{\"question\":\"Email address\",\"fields\":[{\"key\":\"emailAddress\",\"title\":\"Email address\",\"type\":\"text\",\"answer\":\"ali@cautionyourblast.com\"},{\"key\":\"publishEmail\",\"title\":\"Can we publish this email address on GOV.UK?\",\"type\":\"list\",\"answer\":\"noPublish\"}]},{\"question\":\"Email address for GOV.UK\",\"fields\":[{\"key\":\"publicEmailAddress\",\"title\":\"Email address (Public)\",\"type\":\"text\",\"answer\":\"ali@cautionyourblast.com\"}]},{\"question\":\"Phone number\",\"fields\":[{\"key\":\"phoneNumber\",\"title\":\"Phone number\",\"type\":\"text\",\"answer\":\"+4869805499199999\"},{\"key\":\"contactPhoneNumber\",\"title\":\"Emergency / out of hours number\",\"type\":\"text\",\"answer\":\"+48999999999\"}]},{\"question\":\"Full website address (optional)\",\"fields\":[{\"key\":\"websiteAddress\",\"title\":\"Website\",\"type\":\"text\",\"answer\":\"https://tlumaczprzysiegly.waw.pl/XXXX\"}]},{\"question\":\"How do you carry out your services?\",\"fields\":[{\"key\":\"deliveryOfServices\",\"title\":\"Where services can be carried out\",\"type\":\"list\",\"answer\":[\"inPerson\"]}]},{\"question\":\"Which areas of the country do you serve?\",\"fields\":[{\"key\":\"regions\",\"title\":\"Regions covered\",\"type\":\"text\",\"answer\":\"Warsaw and satellite cities XXXX\"}]},{\"question\":\"Which languages do you translate or interpret?\",\"fields\":[{\"key\":\"languagesProvided\",\"title\":\"Language\",\"type\":\"list\",\"answer\":[\"pl\",\"de\"]}]},{\"question\":\"What services do you provide?\",\"fields\":[{\"key\":\"servicesProvided\",\"title\":\"What services do you provide?\",\"type\":\"list\",\"answer\":[\"Translation\",\"Interpretation\"]}]},{\"question\":\"What specialities do you provide translation and interpretation in?\",\"fields\":[{\"key\":\"translationSpecialties\",\"title\":\"Translation services\",\"type\":\"list\",\"answer\":[\"Legal\",\"Medical\",\"Events\",\"Publishing\",\"Business and commercial\",\"Digital media\",\"General translation\"]},{\"key\":\"interpreterServices\",\"title\":\"Interpretation services\",\"type\":\"list\",\"answer\":[\"Medical assistance\",\"Police and local authorities\",\"Courts and legal\",\"Events\",\"Media and communications\",\"Business and commerce\",\"General interpretation \"]}]},{\"question\":\"Are you a certified or sworn translator in your country?\",\"fields\":[{\"key\":\"swornTranslations\",\"title\":\"Are you a certified or sworn translator in your country?\",\"type\":\"list\",\"answer\":true}]},{\"question\":\"Are you a certified or sworn interpreter in your country?\",\"fields\":[{\"key\":\"swornInterpretations\",\"title\":\"Are you a certified or sworn interpreter in your country?\",\"type\":\"list\",\"answer\":true}]},{\"question\":\"Have you provided services to British nationals before?\",\"fields\":[{\"key\":\"representedBritishNationals\",\"title\":\"Have you provided services to British nationals before?\",\"type\":\"list\",\"answer\":false}]},{\"question\":\"Declaration\",\"fields\":[{\"key\":\"declaration\",\"title\":\"Declaration\",\"type\":\"list\",\"answer\":[\"confirm\"]}]}],\"metadata\":{\"type\":\"translators-interpreters\",\"paymentSkipped\":false}}"}
`

Therefore, the issue must be somewhere in the ingest controller / processing